static/i18n/en/app.yml

```diff
@@ -340,31 +342,31 @@ settings:
   softResetModal:
     title: Clear cache
     subTitle: Are you sure?
-    desc: Clearing the Ledger Live cache forces network resynchronization.
+    desc: Clearing the Ledger Live cache forces network resynchronization
   removeAccountModal:
     title: Remove account
     subTitle: Are you sure?
-    desc: The account will no longer be included in your portfolio. Accounts can always be re-added.
+    desc: The account will no longer be included in your portfolio. This operation does not affect your assets. Accounts can always be re-added.
   exportLogs:
     title: Export logs
-    desc: Exporting Ledger Live logs may be necessary for troubleshooting purposes.
+    desc: Exporting Ledger Live logs may be necessary for troubleshooting purposes
     btn: Export
 password:
-  warning_0: Warning 0
-  warning_1: Warning 1
-  warning_2: Warning 2
-  warning_3: Warning 3
-  warning_4: Warning 4
-  errorMessageIncorrectPassword: The password you entered is incorrect.
-  errorMessageNotMatchingPassword: Passwords don't match.
+  warning_0: Very weak
+  warning_1: Weak
+  warning_2: Medium
+  warning_3: Strong
+  warning_4: Very strong
```

Comment:
These are not used

Reply:
Thought they would be in the future, so why not phrase them

Reply:
IMO we need to kill today all wording that are not in the app. 
we should not anticipate future potential feature, because it's gonna be asking more things to translators and we are not sure it's gonna happen.

soon we should do a pass on cleaning up all unused things and also all duplicates ;)